tldw_Server_API/app/api/v1/router_groups/content.py

```
@@ -1,14 +1,986 @@
+        specs.append(RouterSpec(
+            router=media_embeddings_router,
+            prefix=f"{API_V1_PREFIX}",
+            tags=("media-embeddings",),
+            route_key="media-embeddings",
+        ))
+    except Exception as e:  # noqa: BLE001
+        logger.debug(f"Skipping media_embeddings router: {e}")
+
+    # Evaluations and OCR are lazy so route policy can disable them before
+    # importing modules with heavier optional dependencies.
+    def _evaluations_router_factory():
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_evaluations_router_factory`

(ANN202)